recipes/ckanweb-deploy-exts.rb

```diff
@@ -144,10 +153,12 @@
 		end
 
 		if (::File.directory?("#{install_dir}")) then
-			execute "Ensure correct Git origin" do
-				user "#{account_name}"
-				cwd "#{install_dir}"
-				command "git remote set-url origin '#{app['app_source']['url']}'"
+			if app['app_source']['type'].casecmp("git") == 0 then
```

Comment:
No action if the directory exists but it's *not* a Git URL?

Reply:
how can you set a git origin if the type is not git. ie other scm (svn), or s3 or plain python public version.

Reply:
I think trying to handle non-Git URLs would take a fair bit of rework at this point.